python/operator/bot/operator_bot.py

```diff
@@ -21,9 +21,8 @@ class CHAT:
     UserSession = 'Chat.V1:UserSession'
```

Comment:
You changed the case of `SLACK_INTEGRATION`, so I suggest changing the case here too (particularly since you changed the case of a similar file in `user/bot/user_bot.py`).

Separate question that I'm sure you were asking yourself as well, as we're both looking at this with fresh eyes: why are this declaration and the one in `user/bot/user_bot.py`) separate from each other?

Reply:
For the former, I have renamed that in a subsequent commit.

For the latter, I imagine this is due to a lack of precedent for inter directory code sharing between bots for different parties; there is further refactor potential here but I'm inclined to table it for a subsequent PR.